### ISSUE TITLE
[E2E] Timm resnest101e amp_bf16 training accuracy failed

### ISSUE THREAD

**Issue by mengfei25** [intel/torch-xpu-ops#705]
### 🐛 Describe the bug

Model list:
- [ ] `resnest101e`

E0804 04:47:13.347000 901510 torch/_dynamo/utils.py:1558] RMSE (res-fp64): nan, (ref-fp64): 0.00000 and shape=torch.Size([128]). res.dtype: torch.float32, multiplier: 3.000000, tol: 0.010000
E0804 04:47:13.348000 901510 torch/_dynamo/utils.py:1450] Accuracy failed for key name bn1.bias.grad
fail_accuracy

### Versions

torch-xpu-ops: https://github.com/intel/torch-xpu-ops/commit/1d70431c072db889d9a47ea4956049fe340a426d
pytorch: d224857b3af5c9d5a3c7a48401475c09d90db296
device: pvc 1100, bundle: 0.5.3, driver: 803.61

**Comment by retonym** [comment]
low priority for ampbf16 training not included in Meta PyTorch dashboard
